### ISSUE TITLE
bug: false positive "Conditionally disabled code"

### ISSUE THREAD

**Issue by distilledwater105** [intellij-rust/intellij-rust#10632]
## Environment

* **IntelliJ Rust plugin version:** 0.4.197.5401-231
* **Rust toolchain version:** 1.72.0-nightly
* **IDE name and version:** CLion 2023.1.4
* **Operating system:** Arch Linux

## Problem description
The plugin thinks that the code is disabled when it actually isn't.

## Steps to reproduce
```rust
macro_rules! test {
    ($ident:ident, $block:block) => {
        #[test]
        fn $ident() {
            cfg_if::cfg_if! {
                if #[cfg(loom)] {
                    loom::model(|| $block);
                } else {
                    $block;
                }
            }
        }
    };
}

test!(my_test, { // Conditionally disabled code
    assert_eq!(true, true);
});
```
